selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -345,7 +345,7 @@ NetworkType WifiManager::currentNetworkType() {
   return NetworkType::NONE;
 }
 
-void WifiManager::updateGsmSettings(bool roaming, QString apn) {
+void WifiManager::updateGsmSettings(bool roaming, QString apn, bool unmeteredOverride) {
```

Comment:
unmeteredOverride is confusing too. in the theme of simplification, let's also internally just pass around a "metered state".